src/main/java/com/techsophy/tsf/wrapperservice/config/UxControllerCamundaServletContextPath.java

```diff
@@ -0,0 +1,27 @@
+package com.techsophy.tsf.wrapperservice.config;
+
+import com.techsophy.tsf.wrapperservice.utils.TokenUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import static com.techsophy.tsf.wrapperservice.constants.ApplicationConstants.*;
+import static com.techsophy.tsf.wrapperservice.constants.MessageConstants.URL_SEPERATOR;
+
+@Component
+public class UxControllerCamundaServletContextPath {
```

Comment:
add function that should take a relative path as string and return the absolute path to be called